CHANGES.rst

```diff
@@ -1,6 +1,19 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-19)
+--------------------------
+
+- Adds new API for Gitlab integration.
+- Adds new API for Kubernetes ingresses.
+- Adds new APIs for management of user secrets.
+- Adds EOS storage Kubernetes configuration.
+- Adds HTCondor and Slurm compute backends.
+- Adds support for streaming file uploads.
+- Allows unpacked CVMFS and CMS open data volumes.
+- Adds Serial workflow steps.
```

Comment:
Just to clarify, is it partial workflow execution?

Reply:
It's for step name and compute backend, I'll reformulate.